### PR TITLE
LVRUBYM-324: Create Groups controller

### PR DESCRIPTION
**Pull request by JulkaIII**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVRUBYM-204)
* [JIRA subtask](http://ssu-jira.softserveinc.com/browse/LVRUBYM-324)

## Code reviewers

- [ ] @vikabelous
- [ ] @skrypnyk88
- [ ] @yaremakleputs

### Second Level Review

- [ ] @omaik

## Summary of issue

Need to ad gems for testing  into the application

## Summary of change

- added GroupsController
- added controller specs

## Testing approach

Rspec tests

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Comment by skrypnyk88** [comment]
I think the group should belong to teacher. I do not see it here.

File: app/views/v1/groups/index.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1 @@
+json.array! @groups
```
Comment (vikabelous) [inline app/views/v1/groups/index.json.jbuilder]: I think we don't need `updated_at` and `created_at` fields in json

File: spec/controllers/v1/groups_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  end
+
+  describe 'GET #show' do
+    it 'renders group json' do
+      get :show, format: :json,
+                 params: { id: group }
+      expect(response.body).to eq(group_json(group))
+    end
+  end
+
```
Comment (vikabelous) [inline spec/controllers/v1/groups_controller_spec.rb]: No test cases for create action

File: app/views/v1/groups/index.json.jbuilder (hunk 1 above)
Comment (yaremakleputs) [inline app/views/v1/groups/index.json.jbuilder]: you lost new line

File: app/views/v1/groups/show.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.(@group, :id, :name)
```
Comment (yaremakleputs) [inline app/views/v1/groups/show.json.jbuilder]: same

File: app/views/v1/groups/index.json.jbuilder (hunk 1 above)
Comment (JulkaIII) [inline app/views/v1/groups/index.json.jbuilder]: There is empty line :) If it's not, there would be red mark

File: app/views/v1/groups/show.json.jbuilder (hunk 3 above)
Comment (JulkaIII) [inline app/views/v1/groups/show.json.jbuilder]: the same

File: spec/controllers/v1/groups_controller_spec.rb (hunk 2 above)
Comment (JulkaIII) [inline spec/controllers/v1/groups_controller_spec.rb]: done

File: app/views/v1/groups/index.json.jbuilder (hunk 1 above)
Comment (JulkaIII) [inline app/views/v1/groups/index.json.jbuilder]: done

**Pull request closed without merging**